docs/using-airbyte/schema-change-management.md

```diff
@@ -6,100 +6,120 @@ products: all
 
-A connection will always automatically be disabled if an existing primary key or cursor field is removed. You must review and fix the changes before editing the connection or resuming syncs.
+Breaking schema changes occur when:
 
-Breaking changes can also occur when a new major version of the connector is released. In these cases, the connection will alert you of a breaking change but continue to sync until the cutoff date for upgrade. On the cutoff date, the connection will automatically be disabled on that date to prevent failure or unexpected behavior. It is **highly recommended** to upgrade before the cutoff date to ensure you continue syncing without interruption.
+- An existing primary key is removed from the source
+- An existing cursor is removed from the source
 
-A major version upgrade will include a breaking change if any of these apply:
+To re-enable the streams:
 
-| Type of Change                            | Description                                                                                                                     |
-| ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------- |
-| Connector Spec Change                     | The configuration has been changed and syncs will fail until users reconfigure or re-authenticate.                              |
-| Schema Change                             | The type of property previously present within a record has changed and a refresh of the source schema is required.             |
-| Stream or Property Removal                | Data that was previously being synced is no longer going to be synced                                                           |
-| Destination Format / Normalization Change | The way the destination writes the final data or how Airbyte cleans that data is changing in a way that requires a full refresh |
-| State Changes                             | The format of the source’s state has changed, and the full dataset will need to be re-synced                                    |
+1. In the Airbyte UI, click **Connections**. Select a connection and navigate to the **Schema** tab. If schema changes have been detected, you'll see a red "i" icon next to the `Schema` tab.
 
-To review and fix breaking schema changes:
+2. Click the stream to open the field details for that stream. Ensure your cursor and primary key are selected. Changes are automatically saved as soon as the fields are selected.
 
-1. In the Airbyte UI, click **Connections** and select the connection with breaking changes.
+5. Exit the field selection pane and click **Save changes** to ensure the schema changes are applied to the next sync. You will be prompted to refresh or clear the affected streams so that Airbyte can ensure future syncs are successful. Airbyte highly recommends refreshing or clearing streams and sync all historical data again.
```

Comment:
"Changes are automatically saved..." and "click `Save changes` to ensure schema changes are applied" seems confusing.  I know what it's getting at (that the panel itself has no save button) but if all they do is step 2, nothing will _actually_ get saved.

Reply:
Agreed, I removed the auto-save here!